calls/call_000.md

```diff
@@ -0,0 +1,86 @@
+
+Alex Stokes: Working on the Trinity client at the Ethereum Foundation.
+[1:15](https://youtu.be/ot_io5Xi13M?t=76)
+
+Andrei Ivasko: Contributor to the Prysm client and developer at Blockchains LLC.
+[1:42](https://youtu.be/ot_io5Xi13M?t=102)
+
+Trent Van Epps: Protocol testing at WhiteBlock.
+[2:05](https://youtu.be/ot_io5Xi13M?t=125)
+
+Bryant Eisenbach: Ethereum community member.
+[3:00](https://youtu.be/ot_io5Xi13M?t=180)
+
+Danny Ryan: Research at the Ethereum Foundation. Has also worked on a proposal similar to this [EIP-1011](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1011.md).
+[4:40](https://youtu.be/ot_io5Xi13M?t=280)
+
+Danno Ferrin: Works on the Eth1 side of PegaSys at Consensys.
+[5:27](https://youtu.be/ot_io5Xi13M?t=327)
+
+????
+[5:58](https://youtu.be/ot_io5Xi13M?t=358)
+
+## Discussions
+[6:30](https://youtu.be/ot_io5Xi13M?t=390)
+
+Two topics that should be discussed in this call are given:
+1. Approach to consensus change on Eth1. [7:10](https://youtu.be/ot_io5Xi13M?t=430)
+2. Eth1 token issuance reduction. [7:42](https://youtu.be/ot_io5Xi13M?t=462)
+
+### Double Minting
+[8:04](https://youtu.be/ot_io5Xi13M?t=484)
+
+The community is concerned about "minting money in two spots", which ties into the second discussion point. Basically, with both the Eth2 beacon chain and Eth1 POW chain running at the same time, there will be an increased total issuance of ETH. Ideally, the issuance on the Eth1 chain would be reduced using the finality gadget.
+
+### Hardfork Schedule
+[9:18](https://youtu.be/ot_io5Xi13M?t=558)
+
+- The hardfork schedule is in flux and there are different release models under consideration. [9:18](https://youtu.be/ot_io5Xi13M?t=558)
+- A release timeline that adds small changes to the network more frequently is desirable, as opposed to larger releases with less frequency. [11:44](https://youtu.be/ot_io5Xi13M?t=704)
+- Issuance reduction would require a hardfork, however, a finality gadget without an issuance change would not necessarily require a hardfork.  [12:55](https://youtu.be/ot_io5Xi13M?t=775)
+
+### Eth1 Beacon Chain Awareness
+[15:20](https://youtu.be/ot_io5Xi13M?t=920)
+
+- At least two ways to do this i.) have operators run a separate Eth2 client alongside their Eth1 client or ii.) embed an Eth2 client inside of Eth1. [16:35](https://youtu.be/ot_io5Xi13M?t=995)
+- Having each Eth1 node connect to a separate Eth2 node seems like the most straight forward solution. [18:10](https://youtu.be/ot_io5Xi13M?t=1090)
+- Possibility for "herd immunity" where Eth1 nodes that are connected to the beacon chain filter out Eth1 blocks with bad finality information. [18:39](https://youtu.be/ot_io5Xi13M?t=1119)
```

Comment:
```suggestion
- Possibility for "herd immunity" where Eth1 nodes that are connected to the beacon chain filter out Eth1 blocks with bad finality information. Involves the scenario where a third-party/miner/etc. is putting finality data inside the eth1 protocol so that nodes not following the beacon chain can still access it. [18:39](https://youtu.be/ot_io5Xi13M?t=1119)
```